daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -402,14 +396,25 @@ class CLIConf(arguments: Array[String], stdout: PrintStream, stderr: PrintStream
       name = 'T',
       keyName = "tunable",
       valueName = "value",
-      descr = "Tunable configuration options to change Daffodil's behavior"
+      descr = "Tunable configuration options to change Daffodil's behavior. " +
+        "Only valid with the --schema option. Cannot be used with the --parser option."
     )
     val validate: ScallopOption[ValidationMode.Type] = opt[ValidationMode.Type](
       short = 'V',
       default = Some(ValidationMode.Off),
       argName = "mode",
       descr = "Validation mode. Use 'on', 'limited', 'off', or a validator plugin name."
     )
+    val debug = opt[Option[String]](
+      argName = "file",
+      descr =
+        "Enable the interactive debugger. Optionally, read initial debugger commands from [file] if provided. " +
+          "Cannot be used with --trace option. They are mutually exclusive options."
```

Comment:
Remove "they are mutually exclusive options, think "Cannot be used with ---trace option" is sufficient.